src/walletConnect/screens/WalletConnectRequest.tsx

```diff
@@ -53,13 +55,15 @@ function WalletConnectRequest({ route: { params } }: Props) {
 const styles = StyleSheet.create({
   container: {
     padding: Spacing.Thick24,
-    paddingBottom: 0, // SafeAreaView already adds enough space here
-    flex: 1,
+  },
+  loadingTimeoutContainer: {
+    alignItems: 'center',
+    minHeight: 370,
```

Comment:
because the WC connection flow is always connecting -> session request, and while connecting we don't know the height of the session request content, there is a bit of jumping in the height of the bottom sheet. i've attempted to reduce the jump with a min height here but there is still some movement when the session request is displayed as the sheet adjusts to the new content height....

before setting a min height:

https://user-images.githubusercontent.com/20150449/216942255-b0908e98-d71b-4087-8c79-6a0d92269702.mp4



after setting a min height:

https://user-images.githubusercontent.com/20150449/216942286-59cd20d0-9a7e-4888-aeae-07f3ac643a14.mp4

Reply:
Great! 
I wonder if the min height should be smaller, this way it's more likely it goes from small -> actual height which is bigger so it looks slightly nicer?

Alternatively, should we use a skeleton placeholder? out of scope of this PR of course, we could discuss with Reed.

Reply:
hmm, i'm actually not sure that this is a straightforward problem to solve :( 

i haven't found the exact parts of the bottom sheet lib that cause the problem but from observation i think that the animated content height / snap points managed by the bottom sheet lib are long living (they are using `useSharedValue`). after a bottom sheet is mounted / measured, that value persists and is the first value that is used for the next bottom sheet. 

in the video above, there are 3 heights - the first one is from the previous connection bottom sheet, the second one is the minHeight (370) that is set on the loading state, the third one is the height of the sheet with the session request content. i need to spend more time to confirm this theory though

Reply:
i tried adding some logs to the `useBottomSheetDynamicSnapPoints` hook and i think my theory is right - after a bottom sheet is dismissed, the `animatedContentHeight` doesn't get reset to 0 so the next sheets use the last known value. i think this is just because we've got `useBottomSheetDynamicSnapPoints` at the very top level. idk if this is a problem worth solving yet 😅 i think the experience is pretty fine

Reply:
oh yeah right, I feared this would happen with our approach. 
Not the end of the world like you said, would be nice to have a sheet independent solution in the lib itself.